filament/src/PostProcessManager.cpp

```diff
@@ -187,7 +187,8 @@ PostProcessManager::PostProcessMaterial& PostProcessManager::getPostProcessMater
 
 #define MATERIAL(n) MATERIALS_ ## n ## _DATA, MATERIALS_ ## n ## _SIZE
 
-void PostProcessManager::init() noexcept {
+void PostProcessManager::init(bool disableFeedbackLoops) noexcept {
+    mDisableFeedbackLoops = disableFeedbackLoops;
```

Comment:
you don't need to pass the boolean here. You have access to `DriverApi` just below.

Reply:
Gah, I'm so out of it.  Fixed, and squashed.